src/Hepteracts.ts

```diff
@@ -115,7 +115,7 @@ export class HepteractCraft {
         for (const item in this.OTHER_CONVERSIONS) {
             if (typeof player[item as keyof Player] === 'number')
                 (player[item as keyof Player] as number) -= amountToCraft * this.OTHER_CONVERSIONS[item as keyof Player];
-            else if (Object.prototype.isPrototypeOf.call(Cube, player[item as keyof Player]))
+            else if (Object.prototype.isPrototypeOf(Cube))
```

Comment:
needs to be reverted

Reply:
Out of curiosity, where does this cause issues? Unless having the hept crafts not take cubes was intentional, it seemed to work fine in testing.

Reply:
Correct that, I'm stupid and see the issue.